openfl/component/envoy/workspace.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Workspace module."""
+
+import logging
+import os
+import shutil
+import sys
+import time
+from subprocess import check_call
+from sys import executable
+
+logger = logging.getLogger(__name__)
+
+
+class CollaboratorWorkspace:
```

Comment:
I suggest moving this away from the components package too.

Reply:
Moved to utilities/workspace.py